src/NativeScript/JSErrors.mm

```diff
@@ -35,8 +35,55 @@ void TNSSetUncaughtErrorHandler(TNSUncaughtErrorHandler handler) {
 using namespace JSC;
 using namespace Inspector;
 
+Boolean setStackTraceProperty(JSC::ExecState* execState, JSC::Exception* exception, std::string stackTraceContent) {
```

Comment:
You'd better use `bool` instead of `Boolean` - it's the type `putDirect` returns.